policy/templates/subtemplates/auto/api-tests.gotmpl

```diff
@@ -16,7 +16,7 @@
 
       - name: Setup tmate session only in debug mode
         uses: mxschmitt/action-tmate@v3
-        if: {{`${{ runner.debug == '1' }}`}}
+        if: runner.debug == '1
```

Comment:
Shouldn't there be a closing single quote?

Reply:
Didn't the validator or the formatter catch this?

Reply:
This should be wrapped in ${{}}, no?

Reply:
I was wondering about that too, but  it seems statements in `if:` is automatically treated as expressions, so the wrapping is not  necessary unless they start with a `!`.

> When you use expressions in an if conditional, you can, optionally, omit the ${{ }} expression syntax because GitHub Actions automatically evaluates the if conditional as an expression. However, this exception does not apply everywhere.

from https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif